wikitemplate-android.md

```diff
@@ -43,32 +46,37 @@ Pre-Requisite: Put previous build shortcut on home screen. Also have several sit
 ## Find in page
 
 - [ ] Ensure search box is shown when selected via the hamburger menu
-- [ ] Test successful find
-- [ ] Test forward and backward find navigation
-- [ ] Test failed find shows 0 results
+- [ ] Verify successful find
+- [ ] Verify forward and backward find navigation
```

Comment:
nit: Verify that the forward and backward navigation icons under find are working